docs/moonray/howto/networking/default-ingress-mr.md

```diff
@@ -0,0 +1,116 @@
+# How to use default Ingress
+
+{{product}} allows you to configure Ingress into your cluster. When
+enabled, it tells your cluster how external HTTP and HTTPS traffic should be
+routed to its services.
+
+## What you'll need
+
+This guide assumes the following:
+
+- You have root or sudo access to the machine
+- You have a bootstrapped {{product}} cluster (see the [Getting
+  Started][getting-started-guide] guide).
+
+## Check Ingress status
+
+Find out whether Ingress is enabled or disabled with the following command:
+
+```
+sudo k8s status
+```
+
+The default state for the cluster is `ingress disabled`.
```

Comment:
maybe we can change this to something like below, so that it won't get out dated.
```md
Make sure the ingress is enabled.
```